packages/celotool/src/lib/generate_utils.ts

```diff
@@ -194,6 +201,8 @@ export const generateGenesis = ({
     genesis.extraData = generateIstanbulExtraData(validators)
     genesis.config.istanbul = {
       policy: 0,
+      period: blockTime,
+      requesttimeout: requestTimeout,
```

Comment:
Is this currently used in the genesis block? a quick grep through the codebase only revealed the cmdline parameter

Reply:
my thinking was in future we will wants 3rd party nodes to be able to run without specifying these via the command line. (in fact, it doesn't even really make sense for blockTime to be a cmd line parameter)